web/src/test/java/org/springframework/security/web/server/csrf/CookieServerCsrfTokenRepositoryTests.java

```diff
@@ -61,78 +61,88 @@ public class CookieServerCsrfTokenRepositoryTests {
 
 	private String expectedCookieValue = "csrfToken";
 
+	private String expectedSameSitePolicy = null;
+
+
 	@BeforeEach
 	public void setUp() {
 		this.csrfTokenRepository = new CookieServerCsrfTokenRepository();
 		this.request = MockServerHttpRequest.get("/someUri");
 	}
 
 	@Test
-	public void generateTokenWhenDefaultThenDefaults() {
```

Comment:
Thank you for cleaning these up.

To ensure that each commit stays focused on an atomic change, will you please move these Junit 5 cleanup items to a separate commit in this PR?

Reply:
Sure. no problem.